packages/react-query/src/__tests__/suspense.test-d.tsx

```diff
@@ -121,4 +123,16 @@ describe('useSuspenseInfiniteQuery', () => {
     // @ts-expect-error TS2339
     query.isPlaceholderData
   })
+
+  it('should not accept skipToken type for queryFn in useSuspenseQuery', () => {
+    doNotExecute(() => {
+      const query: UseSuspenseQueryOptions = {
+        // @ts-expect-error
+        queryFn: skipToken,
+        queryKey: [1],
+      }
+
+      return query
+    })
+  })
```

Comment:
*.test-d.tsx file don't be excuted by vitest type test. it just type check by default
https://vitest.dev/guide/testing-types

```suggestion
  it('should not accept skipToken type for queryFn in useSuspenseQuery', () => {
    const query: UseSuspenseQueryOptions = {
      // @ts-expect-error
      queryFn: skipToken,
      queryKey: [1],
    }

    return query
  })
```

Reply:
Agreed, I will remove it! thx